### ISSUE TITLE
Is test.list used as cross validation set or test set?

### ISSUE THREAD

**Issue by maxxfan** [PaddlePaddle/Paddle#495]
A quote from 
http://www.paddlepaddle.org/doc/ui/data_provider/index.html
"Testing or evaluating will not be performed during training if the test.list is not set or set to None. Otherwise, PaddlePaddle will evaluate the trained model by the specified tesing data while training, every testing period (a user defined command line parameter in PaddlePaddle) to prevent over-fitting."

Question: Still not so clear about how test data is used. Is the data in test.list used as cross validation data? Or just used as test error calculation(not affect training/model)? Thanks.

**Comment by luotao1** [comment]
The data in test.list is just used as  test error calculation, not affect training model.
